### ISSUE TITLE
What version of Fourier Neural Operator is used in the MWT paper?

### ISSUE THREAD

**Issue by scaomath** [gaurav71531/mwt-operator#1]
Hi, thanks for the sharing the nicely written code on the wavelet neural operator.

I wonder what version of FNO is used in the comparison? for example in Table 2 of https://arxiv.org/abs/2109.13459v1

Because in Table 2, FNO1d still has only `1e-2` evaluation relative error for the Burgers' benchmark. 

In the new implementation, if `BatchNorm` is removed, and change the ReLU activation to GeLU or SiLU, 500 epochs of MultiStepLR scheduling will yield about `2.5e-3` to `2.7e-3` evaluation relative error: see zongyi-li/fourier_neural_operator#30 , even with a 1cycleLR + 100 epochs, the evaluation error is around `4.1e-3`.

**Comment by gaurav71531** [comment]
For Burgers' eqn the results are in Table-7 where after using 3 seeds we get average error ~3.33e-3. We have used Pytorch 1.8 cu 11.0

**Comment by scaomath** [comment]
Thanks.
